Library/DiscUtils.Ntfs/Bitmap.cs

```diff
@@ -201,6 +201,16 @@ public byte GetByte(long index)
             return 0;
         }
 
+        public int GetBytes(byte[] buffer, long index, int count)
```

Comment:
Either:
* make it private/internal 
* change it so that it takes an offset for the buffer as well (like `Array.Copy()`)
* some other way, so that we keep within the observed standards in .NET API's .. :)